vscode/src/notebookTemplate.ts

```diff
@@ -11,7 +11,7 @@ export const notebookTemplate: vscode.NotebookData = {
     {
       kind: vscode.NotebookCellKind.Markup,
       languageId: "markdown",
-      value: `## Azure Quantum Q# notebook
+      value: `## Microsoft Quantum Q# notebook
```

Comment:
Same as above - the notebook content does include Azure Quantum so the name may be fine?